class/cert-manager.yml

```diff
@@ -1,11 +1,42 @@
 parameters:
+  =_config:
+    # --- Exoscale Webhook
+    exoscale_webhook_values:
+      'True':
+        input_paths:
+          - ${_base_directory}/component/helm/exoscale-webhook.jsonnet
+        input_type: jsonnet
+        output_path: ${_base_directory}/helmcharts/exoscale-webhook/${cert_manager:charts:exoscale_webhook:version}/
+      'False': &option_false
```

Comment:
Having an "empty" jsonnet compile step isn't very nice, but the alternative isn't nice either (having a dummy base `compile` list and extending it when deploying the exoscale webhook)